### ISSUE TITLE
[Accessibility]: Monaco editor has an invalid role attribute

### ISSUE THREAD

**Issue by kopancek** [sourcegraph/sourcegraph#35715]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34120

### Problem description

When using monaco editor for editing JSON settings, it adds a `div` element with `role="code"`. However this is an invalid value for the `role` attribute based on ARC Toolkit check.
<img width="294" alt="Screenshot 2022-05-19 at 17 36 22" src="https://user-images.githubusercontent.com/9974711/169338215-45a28f56-5aac-4e78-8ed2-1f32c62f02a4.png">

### Expected behavior

Valid `role` attribute or no `role` attribute is used in monaco editor

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.
